dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -45,14 +45,14 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
 
   test "authorizing with unknown clever teacher account" do
     auth = stub('auth',
-                uid: '1111',
-                provider: 'clever',
-                info: stub('info', nickname: '',
-                           name: {'first' => 'Hat', 'last' => 'Cat'},
-                           email: 'first_last@clever_teacher.xx',
-                           user_type: 'teacher',
-                           dob: nil,
-                           gender: nil))
+      uid: '1111',
+      provider: 'clever',
+      info: stub('info', nickname: '',
+                 name: {'first' => 'Hat', 'last' => 'Cat'},
```

Comment:
Yeah, it should.  Maybe Rubocop is getting confused with the nested method calls?  I'll fix by hand.